### ISSUE TITLE
Steel Lock Pick Uncraftable

### ISSUE THREAD

**Issue by WenXin20** [Melonslise/Locks#89]
`Minecraft v1.16.5`
`Forge v36.1.16`
`Locks v3.0.3`

The recipe for the Steel Lock Pick instead crafts the Iron Lock Pick.

![2021-05-28_00 30 37](https://user-images.githubusercontent.com/12000593/119934789-fa6f9080-bf4b-11eb-940a-7aa768b2412c.png)

**Comment by Darkmega18** [comment]
Saw this also. recipe crafts the wrong thing. I'd also like to suggest in here, in relation to crafting that the pick crafting recipes are potentially a bit steep when it comes to the metallic ones and should probably either craft atleast 4-8 for that many resources. (if not, I'm guessing I can datapack edit that myself, yes?)

**Comment by alvaxdro** [comment]
I found the solution in the mod's files:
1.open the mod in winrar
2.go to data;locks;recipes
3.find the file''steel_lock_pick.json'' and open it in note pad(dragging the file in)
4.this will come out:{
	"type": "minecraft:crafting_shaped",
	"pattern":
	[
		" SS",
		" S ",
		"S  "
	],
	"key":
	{
		"S":
		{
			"tag": "forge:ingots/steel"
		}
	},
	"result":
	{
		"item": "locks:iron_lock_pick",
		"count": 2
	}
}
change ''locks:iron_lock_pick'' to ''locks:steel_lock_pick''
5.save it and drag it to the win rar(choose''agree'')
6.close it and it's done

**Comment by WenXin20** [comment]
You can just create a datapack with the new recipe, no need to edit the jar file.

**Comment by alvaxdro** [comment]
> You can just create a datapack with the new recipe, no need to edit the jar file.


also worth

**Comment by Darkmega18** [comment]
yeah, I already did it myself in my modpack. but figured not everyones suchly inclined to fiddle with this stuff, nor would they know how.

**Comment by WenXin20** [comment]
> yeah, I already did it myself in my modpack. but figured not everyones suchly inclined to fiddle with this stuff, nor would they know how.

It's pretty simple once set up, similar to a resource pack.
https://minecraft.fandom.com/wiki/Tutorials/Creating_a_data_pack

**Comment by Darkmega18** [comment]
for other people I guess. I said I already know. :V

**Comment by AtlasEra** [comment]
> I found the solution in the mod's files: 1.open the mod in winrar 2.go to data;locks;recipes 3.find the file''steel_lock_pick.json'' and open it in note pad(dragging the file in) 4.this will come out:{ "type": "minecraft:crafting_shaped", "pattern": [ " SS", " S ", "S " ], "key": { "S": { "tag": "forge:ingots/steel" } }, "result": { "item": "locks:iron_lock_pick", "count": 2 } } change ''locks:iron_lock_pick'' to ''locks:steel_lock_pick'' 5.save it and drag it to the win rar(choose''agree'') 6.close it and it's done

Whenever i try and save the edits i made to the .json file, it removes the entire mod .jar file

**Comment by WenXin20** [comment]
> > I found the solution in the mod's files: 1.open the mod in winrar 2.go to data;locks;recipes 3.find the file''steel_lock_pick.json'' and open it in note pad(dragging the file in) 4.this will come out:{ "type": "minecraft:crafting_shaped", "pattern": [ " SS", " S ", "S " ], "key": { "S": { "tag": "forge:ingots/steel" } }, "result": { "item": "locks:iron_lock_pick", "count": 2 } } change ''locks:iron_lock_pick'' to ''locks:steel_lock_pick'' 5.save it and drag it to the win rar(choose''agree'') 6.close it and it's done
> 
> Whenever i try and save the edits i made to the .json file, it removes the entire mod .jar file

Make a [datapack](https://minecraft.wiki/w/Tutorials/Creating_a_data_pack) instead. Editing the jar file can cause this to happen.
